catalogue/webapp/pages/images.tsx

```diff
@@ -0,0 +1,283 @@
+                    aggregations={undefined}
+                  />
+                )}
+              </div>
+            </div>
+          </div>
+        </Space>
+
+        {results?.type === 'ResultList' && results.results.length > 0 && (
+          <>
+            <Space v={{ size: 'l', properties: ['padding-top'] }}>
+              <div className="container">
+                <div className="grid">
+                  <div
+                    className={classNames({
+                      [grid({ s: 12, m: 12, l: 12, xl: 12 })]: true,
+                    })}
+                  >
+                    <div className="flex flex--h-space-between flex--v-center">
+                      <ImagesPagination
+                        query={query}
+                        page={page}
+                        results={results}
+                        imagesRouteProps={imagesRouteProps}
+                        setSavedSearchState={setSavedSearchState}
+                      />
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </Space>
+
+            <Space
+              v={{
+                size: 'l',
+                properties: ['padding-top'],
+              }}
+              style={{ opacity: loading ? 0 : 1 }}
+            >
+              <div className="container">
+                {images && images.type !== 'Error' && (
+                  <ImageEndpointSearchResults
+                    images={images}
+                    apiProps={apiProps}
+                  />
+                )}
+              </div>
+
+              <Space
+                v={{
+                  size: 'l',
+                  properties: ['padding-top', 'padding-bottom'],
+                }}
+              >
+                <div className="container">
+                  <div className="grid">
+                    <div
+                      className={classNames({
+                        [grid({ s: 12, m: 12, l: 12, xl: 12 })]: true,
+                      })}
+                    >
+                      <div className="flex flex--h-space-between flex--v-center">
+                        <ImagesPagination
+                          query={query}
+                          page={page}
+                          results={results}
+                          imagesRouteProps={imagesRouteProps}
+                          setSavedSearchState={setSavedSearchState}
+                        />
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </Space>
+            </Space>
+          </>
+        )}
+      </CataloguePageLayout>
+    </>
+  );
+};
+
+Images.getInitialProps = async (ctx: NextPageContext): Promise<Props> => {
+  const params = ImagesRoute.fromQuery(ctx.query);
+  const apiProps = imagesRouteToApiUrl(params);
+  const hasQuery = !!(params.query && params.query !== '');
+  const imagesOrError = hasQuery
```

Comment:
This shouldn't pass the type check which isn't nullable.
I also think we're moving towards using undefined as our empty value.